libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -436,9 +526,8 @@ export class SDSAutocompleteSearchComponent implements ControlValueAccessor {
    * highlights the index being hovered
    * @param index
    */
-  listItemHover(index: number): void {
-    this.highlightedIndex = index;
-    this.setHighlightedItem(this.results[this.highlightedIndex]);
+  listItemHover(): void {
+    this.highlightedItem[this.HighlightedPropertyName] = false;
```

Comment:
I'm not sure if this is causing it, but when you keyboard navigate and have an item selected, and then you hover the list with the mouse, it stopped highlighting the keyboard selected value. 

When mouse leaves the list we should highlight the keyboard selected value again, or always show the keyboard selected value even when hovering with the mouse.